docs/en/Use-ShardingJDBC-as-storage-implementor.md

```diff
@@ -0,0 +1,18 @@
+# Supported Database Sharding Storage
+Beside the default Elasticsearch storage, it also support the database sharding storage, it allow the users to store data in multiple databases.
```

Comment:
Beside the default Elasticsearch storage, the user can use shardingJDBC and MySQL as storage implementor.

Reply:
OK,I will modify